src/o365/spfx/commands/project/project-externalize/model/IVisitationResult.ts

```diff
@@ -0,0 +1,7 @@
+import { FileEditSuggestion } from "./FileEditSuggestion";
+import { ExternalizeEntry } from "./ExternalizeEntry";
+
+export interface IVisitationResult {
```

Comment:
In the CLI we don't prefix interfaces with `I`, so we should skip it here for consistency

Reply:
removed the I